modules/monitoring-example-service-endpoint-authentication-settings.adoc

```diff
@@ -0,0 +1,152 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="example-service-endpoint-authentication-settings_{context}"]
+= Example service endpoint authentication settings
+
+You can configure authentication for service endpoints for user-defined project monitoring by using `ServiceMonitor` and `PodMonitor` custom resource definitions (CRDs). 
+
+The following samples show different authentication settings for a `ServiceMonitor` resource.
+Each sample shows how to configure a corresponding `Secret` object that contains authentication credentials and other relevant settings.
+
```

Comment:
I think each of these examples should be separate. The rendered preview with the text inside the `====` looks odd. 

```
The following sample shows....
.Sample blah
[source,yaml]
----
apiVersion: v1
```

Reply:
Hello @mburke5678 I have tried many ways to make this as easy to read as possible. I find that this method is actually the best, let me show the other cases:

1. Without a heading, it is just many blacks of texts which makes it really hard to navigate. It would be 6 different blocks of text, which is, in my opinion, not the best solution.
![image](https://github.com/openshift/openshift-docs/assets/98941675/f88add8a-4230-46a5-8a0a-b8cacb3417e3)


2. I tried to create headings for each example. This solution works much better, however, I still think the solution with blocks helps it even more to distinguish:
![image](https://github.com/openshift/openshift-docs/assets/98941675/19663511-8ebb-4f2a-b03f-477bdb671d9a)

3. If I combine the previous two solutions, I will get  the following result. I like this solution the most out of these three.
![image](https://github.com/openshift/openshift-docs/assets/98941675/cef8d268-6581-4217-a610-84fdd2bdc6ca)


I took into concideration one more factor though, and that is that we already have one solution like this in the Monitoring docs: [Remote write](https://docs.openshift.com/container-platform/4.15/observability/monitoring/configuring-the-monitoring-stack.html#example-remote-write-authentication-settings_configuring-the-monitoring-stack) That is why I chose that solution in the end, to have some consistency in the docs.

WDYT? I still like the current solution the most, but the solution no. 3 could also work, however, it would not be as consistent with our other similar example reference doc.

Reply:
@eromanova97 I like #3 also. I will defer to your opinion what works best for our customers and the docs.